### PR TITLE
Changed color of heading on welcome page

### PR DESCRIPTION
**Pull request by gr8shivam**
#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

#### Short description of what this resolves:
Resolves the change of color of heading on welcome page. (#690)

#### Changes proposed in this pull request:

- Add a class "main-heading" to the heading
- Add a style for the heading in welcome.scss

Fixes a part of #690

### CODE REVIEW

File: app/styles/partials/welcome.scss [hunk 1]
```diff
@@ -26,3 +26,7 @@
     }
   }
 }
+
+.main-container .main-heading{
```
Comment (geekyd) [inline app/styles/partials/welcome.scss]: @gr8shivam Why do we need `main-container` here? Why are we adding the color to `main-container`?

File: app/styles/partials/welcome.scss [hunk 2]
```diff
@@ -26,3 +26,7 @@
...
+  color:#2185d0;
```
Comment (geekyd) [inline app/styles/partials/welcome.scss]: Add space after `:`

**Comment by sumedh123** [comment]
@gr8shivam Also please squash your commits when requested changes are done and write commit message something like "Change color heading on welcome page."

File: app/styles/partials/welcome.scss (hunk 1 above)
Comment (gr8shivam) [reply app/styles/partials/welcome.scss]: We aren't adding color to the main-container. It's css-child targeting. It's just to make sure that main-heading inside the main-container is targeted, so it doesn't conflict with any other element outside main-container that may have a main-heading class.

Comment (gr8shivam) [reply app/styles/partials/welcome.scss]: Waiting for your reply on this. @geekyd

File: app/templates/index.hbs [hunk 3]
```diff
@@ -1,6 +1,6 @@
 <div class="ui container">
```
Comment (geekyd) [inline app/templates/index.hbs]: Add `main-heading` class of `Call For Speakers` also.

File: app/styles/partials/welcome.scss (hunk 1 above)
Comment (geekyd) [reply app/styles/partials/welcome.scss]: @gr8shivam My bad but we are not using the main-heading class anywhere else in the project so it is safe to apply css to a single class(main-heading) . Also please add a space before the starting brace it is leading to failure of es-lint test for this file. Check https://travis-ci.org/fossasia/open-event-frontend/builds/287866115#L1453

File: app/templates/index.hbs (hunk 3 above)
Comment (gr8shivam) [reply app/templates/index.hbs]: Done.

**Comment by geekyd** [comment]
@gr8shivam Please squash all your commits

**Comment by niranjan94** [comment]
Deployment link: https://pr-890-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-890-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-890-open-event-frontend-fossasia.surge.sh

**Comment by niranjan94** [comment]
Deployment link: https://pr-890-open-event-frontend-fossasia.surge.sh

**Comment by gr8shivam** [comment]
Squashed commits.

**Comment by geekyd** [comment]
@gr8shivam Thanks for the PR

**Pull request merged**